### ISSUE TITLE
Error: Context is not a class, it's a struct

### ISSUE THREAD

**Issue by wout** [luckyframework/dexter#29]
I'm getting the following build error on Crystal 0.35:

```crystal
In lib/dexter/src/dexter/log/context.cr:1:7

 1 | class Log::Context

           ^

Error: Context is not a class, it's a struct

The command "crystal spec" exited with 1.
```

https://travis-ci.org/github/tilishop/lucky_svg_sprite.cr/jobs/690646879

Should I ignore it?

**Comment by jwoertink** [comment]
I think Dexter will need to change again for next release just because the last release didn't have built in named tuple or something, but now it does.

**Comment by paulcsmith** [comment]
That is correctly. The nightly/master branch of Crystal has a *bunch* of changes to logging that will require changes to Dexter. Let's leave this open until I fix it though. My plan is (if possible) to release a patch release with compatibility for both so it is easier to upgrade. But a lot changed so we'll see if that's possible!

**Comment by confact** [comment]
Would love this to be fixed as 0.35 is out now :)
